servers/publikator/.env.example

```diff
@@ -7,8 +7,7 @@ CORS_WHITELIST_URL=http://localhost:3005
 #ENGINE_API_KEY=
 
 # default settings for mailchimp campaign creation
-MAILCHIMP_FROM_NAME=Republik
-MAILCHIMP_REPLY_TO=kontakt@republik.ch
+#MAILCHIMP_CAMPAIGN_CONFIGS='[{"key":"org/some-format","from_name":"Republik"}]'
```

Comment:
ENVs are confusing already, so don't repeat the config from `.env.example` here. I'm for the option of only setting it in `.env.example`

Reply:
So true. Changing.